client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -1,34 +1,55 @@
+import { alpha } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
-
 const useStyles = makeStyles((theme) => ({
   form: {
-    width: '100%', // Fix IE 11 issue.
+    width: '100%',
     marginTop: theme.spacing(1),
+    '& .MuiFormControl-root': {
+      marginTop: theme.spacing(1),
+    },
+  },
+  caption: {
+    fontWeight: 'bold',
+  },
+  link: {
+    textDecoration: 'none',
+    color: theme.palette.primary.main,
   },
   label: {
-    fontSize: 19,
-    color: 'rgb(0,0,0,0.4)',
-    paddingLeft: '5px',
+    fontWeight: 800,
+    fontSize: 12,
```

Comment:
I know you are using font sizes consistently but try to use Typography variants instead to maintain font size consistency through the app :)

Ref: https://mui.com/api/typography/

We have over 12 variants and if you give them custom font sizes in the theme, I think that should be more than sufficient for usage across the app 😄